config/hub/samples/clustermanager_v1alpha1_multiclusterworkload.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: clustermanager.onprem.openshift.io/v1alpha1
+kind: MultiClusterWorkload
+metadata:
+  name: multiclusterworkload-sample
+spec:
+  bagOfYAML: |-
```

Comment:
> Thanks @sohankunkerkar ! Is there a reason to create the example yaml for the CRs instead of just the API type definitions themselves?

I was planning to create the API type definitions for these two CRs, but @abhat suggested me to create the example yaml for the CRs instead.